### ISSUE TITLE
Does the form file can't be disabled?

### ISSUE THREAD

**Issue by thepra** [jgthms/bulma#3291]
This is about **Bulma**.

Is it a bug/feature/question or do you need help?
It's a question

### Overview of the problem

This is about the Bulma **CSS framework**

I'm using Bulma **version** [0.9.1]
My **browser** is: Firefox

I am sure this issue is **not a duplicate**?
Yes

### Description

Is there any way to disable property a [bulma form file](https://bulma.io/documentation/form/file/)? From the documentation the isn't any notion, and I did also try to apply in different places the `disabled="disabled"` without much success, the `file-icon` and `file-label` prevail despite setting disabled everywhere.

### Expected behavior

Being able to disable a form file from clicking/interacting

### Actual behavior

No way to set it properly

**Comment by iamarpitpatidar** [comment]
huh.. 
I dunno what you did mate, but its easy, just add disabled attribute to file input and that's it.
```
<div class="file">
  <label class="file-label">
    <input class="file-input" type="file" name="resume" disabled> <--
    <span class="file-cta">
      <span class="file-icon">
        <i class="fas fa-upload"></i>
      </span>
      <span class="file-label">
        Choose a file…
      </span>
    </span>
  </label>
</div>
```

**Comment by thepra** [comment]
It does indeed disable the click but visually the button appears as clickable while I was expecting the same as in other buttons, which when set disable they are greyed out and the cursor changes.

**Comment by iamarpitpatidar** [comment]
Ooh.. i see
need to open a PR

Arpit Patidar | COO - RuneBanks
https://runebanks.com

________________________________

**Comment by iamarpitpatidar** [comment]
@thepra this will fix it.
commit: https://github.com/jgthms/bulma/pull/3293/commits/9055df80a945b67756afc8cfec5c9cd246187a8a

if u need css add these styles
```css
.file.is-primary .file-input[disabled] ~ .file-cta:hover, 
.file.is-primary .file-input[disabled] ~ .file-cta.is-hovered, 
.file.is-primary .file-input[disabled] ~ .file-cta:focus, 
.file.is-primary .file-input[disabled] ~ .file-cta.is-focused, 
.file.is-primary .file-input[disabled] ~ .file-cta:active, 
.file.is-primary .file-input[disabled] ~ .file-cta.is-active {
	background-color: whitesmoke;
	border-color: whitesmoke;
	color: #7a7a7a;
}
.file-input[disabled] ~ .file-cta,
.file-input[disabled] ~ .file-name {
	background-color: whitesmoke;
	border-color: whitesmoke;
	color: #7a7a7a;
}

.file-input[disabled] ~ .file-cta,
.file-input[disabled] ~ .file-name,
.file-input[disabled] ~ .file-cta .file-label {
	cursor: not-allowed;
}

**Comment by thepra** [comment]
Thanks, will be enough for now :)
Should I leave this open until an official release?

**Comment by iamarpitpatidar** [comment]
> Should I leave this open until an official release?
maybe atleast untill the commit's merged

**Comment by thepra** [comment]
Is it fixed on 0.9.2? I don't a documentation update about.

**Comment by iamarpitpatidar** [comment]
nope i don't think so, noone even commented on my PR yet @jgthms 
commit: https://github.com/jgthms/bulma/pull/3293/commits/9055df80a945b67756afc8cfec5c9cd246187a8a
